src/lexer.l

```diff
@@ -73,6 +73,7 @@ oct_esc  [0-7]{1,3}
 }
 
 bpftrace|perf|raw       { return Parser::make_STACK_MODE(yytext, loc); }
+^config                 { return Parser::make_CONFIG(loc); }
```

Comment:
Maybe allow leading whitespace too? This would make config blocks whitespace sensitive which (IIUC) is a departure from all other syntax